### PR TITLE
#1861, adding the SasView version number

### PR DESCRIPTION
**Pull request by Caddy-Jones**
Attempting to solve issue #1861

### CODE REVIEW

File: src/sas/qtgui/MainWindow/GuiManager.py [hunk 1]
```diff
@@ -100,9 +103,12 @@ def __init__(self, parent=None):
         handler.messageWritten.connect(self.appendLog)
 
         # Log the start of the session
-        logging.info(" --- SasView session started ---")
+        version = f"{SASVIEW_VERSION.split('.')[0]}.{SASVIEW_VERSION.split('.')[1]}"
```
Comment (krzywon) [inline src/sas/qtgui/MainWindow/GuiManager.py:106]: This only gives the major and minor version information, but not which point release is being used (5.0 vs 5.0.5). The full version is given in the next line, but this seems unnecessary.

Perhaps combine the two lines:
`logging.info(f" --- SasView session started: v{SASVIEW_VERSION}, {SASVIEW_RELEASE_DATE} ---")`

**Review by krzywon — COMMENTED** [review]
This only does half of what is asked for in #1861 and does not touch on the window title. I took a quick look at changing the window title and that is not as straight forward as I would have expected, so I can't fault you there. I'll update the ticket with what I know.

File: src/sas/qtgui/MainWindow/GuiManager.py (hunk 1 above)
Comment (rozyczko) [reply src/sas/qtgui/MainWindow/GuiManager.py:106]: Adding something like this to MainWindow.py somewhere in `__init__.py` should do the trick
```
        from sas.sasview import __version__ as SASVIEW_VERSION
        self.setWindowTitle("SasView " + str(SASVIEW_VERSION))
```

Comment (rozyczko) [reply src/sas/qtgui/MainWindow/GuiManager.py:106]: it does.

Untitled

**Comment by rozyczko** [comment]
Adding something like this to MainWindow.py somewhere in `__init__.py` should do the trick
```
        from sas.sasview import __version__ as SASVIEW_VERSION
        self.setWindowTitle("SasView " + str(SASVIEW_VERSION))
```

Untitled

**Comment by krzywon** [comment]
@rozyczko - I had tried that but I added `self.setWindowTitle("SasView " + str(SASVIEW_VERSION))` before `self.setupUi(self)` the latter, I now see, resets the window title. Yes, it should be that easy!

**Comment by smk78** [comment]
Ooo, it would be nice to get Mantid to do that too! :-)

**Review by rozyczko — COMMENTED** [review]
This looks good to me!

**Review by butlerpd — APPROVED** [review]
Looks good.  as discussed at the SasView fortnightly meeting this should now be good to go, so approving it now.

**Pull request merged**